indexing-service/src/test/java/org/apache/druid/indexing/common/task/KillUnusedSegmentsTaskTest.java

```diff
@@ -99,8 +103,9 @@ public void testKill() throws Exception
         newSegment(Intervals.of("2019-04-01/2019-05-01"), version)
     );
 
-    Assert.assertEquals(2L, task.getNumBatchesProcessed());
-    Assert.assertEquals(1, task.getNumSegmentsKilled());
+    KillTaskReport.Stats stats = getReportedStats();
+    Assert.assertEquals(2, stats.getNumBatchesProcessed());
```

Comment:
nit: could create an expected KillTaskReport.Stats object, and check for equality in one call, rather than for each member. Similar for other tests.

Reply:
Done. Thanks for the suggestion, the assertions are cleaner now.